### PR TITLE
Adding May Update

### PR DESCRIPTION
**Pull request by duelinggalois**

### CODE REVIEW

**Comment by mdonnellyli** [comment]
- Site looks the same in Preview as the last one. Nothing broke.
- Update content looks good.

Update LGTM

File: docs/updates/May_2024.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+### Contracts
+
+It has been a pleasure to see the team that we have grown into continue to come
+together and become what will be a power house of DeFi development. I have been
+working mostly alone on development of the theory of Ammalgam for years. Now we
+have two deep mathematical minds on the team breaking down my big ideas into
+smaller concepts that can be easily understood and coded. This has been a breath
+of fresh air. I feel confident in saying we have successfully transitioned two
+front engineers to making meaningful contributions to our contracts.
```
Comment (cirethic) [inline docs/updates/May_2024.md:27]:
```suggestion
front-end engineers to making meaningful contributions to our contracts.
```

**Review by cirethic — CHANGES_REQUESTED** [review]
just a tiny tweak

**Review by eswarasai — APPROVED** [review]
LGTM 👍

**Pull request merged**